File: profiles/darling/hud/sysmesboxdisplay.js
var sheet = "Data";
var name = "SysMesBox";

root.SysMesBoxDisplay = {
	Type: SysMesBoxType.Darling,
    DrawType: DrawComponentType.SystemMessage,
	BoxX: 602,
	BoxY: 224,
	BoxMiddleBaseX: 604,
	BoxMinimumWidth: 768,
	BoxMiddleBaseWidth: 434,
	BoxRightBaseWidth: 24,
	BoxRightRemainPad: 48,
	TextFontSize: 32,
	TextMiddleY: 236,
	TextX: 640,
	TextLineHeight: 34,
	TextMarginY: 14,
    ChoicePadding: 40,
    ChoiceY: 365,
    ChoiceXBase: 680,
    MinMaxMesWidth: 294,
    AnimationSpeed: 55,
    FadeInDuration: 0.33,
    FadeOutDuration: 0.25
};

root.Sprites["SelectionHighlight"] = {
    Sheet: "Data",
    Bounds: { X: 65, Y: 33, Width: 94, Height: 30 }
};
root.SysMesBoxDisplay.SelectionHighlight = "SelectionHighlight";

root.Sprites[name + "BoxPartLeft"] = {
    Sheet: sheet,
    Bounds: {
        X: 1,
        Y: 842,
        Width: 48,
        Height: 182
    }
};
root.SysMesBoxDisplay.BoxPartLeft = name + "BoxPartLeft";

root.Sprites[name + "BoxPartRight"] = {
    Sheet: sheet,
    Bounds: {
        X: 483,
        Y: 842,
        Width: 48,
        Height: 182
    }
};
root.SysMesBoxDisplay.BoxPartRight = name + "BoxPartRight";

root.Sprites[name + "BoxPartMiddle"] = {
    Sheet: sheet,
    Bounds: {
        X: 49,
        Y: 842,
        Width: 434,
        Height: 182
    }
};
root.SysMesBoxDisplay.BoxPartMiddle = name + "BoxPartMiddle";

root.Sprites[name + "BoxDecoration"] = {
    Sheet: sheet,
    Bounds: {
        X: 1383,
        Y: 117,
        Width: 592,
        Height: 8
    }
};
root.SysMesBoxDisplay.BoxDecoration = name + "BoxDecoration";

File: profiles/mo8/hud/sysmesboxdisplay.lua
local sheet = "Data";
local name = "SysMesBox";

root.SysMesBoxDisplay = {
    Type = SysMesBoxType.CHLCC,
    DrawType = DrawComponentType.SystemMessage,
    BoxX = 383,
    BoxY = 366,
    TextFontSize = 42,
    TextMiddleY = 412,
    TextX = 942,
    TextLineHeight = 34,
    TextMarginY = 14,
    ChoicePadding = 60,
    ChoiceY = 647,
    ChoiceXBase = 1211,
    MinMaxMesWidth = 294,
    MinHighlightWidth = 48,
    HighlightBaseWidth = 144,
    HighlightYOffset = 12,
    HighlightXOffset = 0,
    HighlightXBase = 658,
    HighlightXStep = 132,
    HighlightRightPartSpriteWidth = 24,
    AnimationSpeed = 55,
    FadeInDuration = 0.33,
    FadeOutDuration = 0.25
};

root.Sprites[name .. "Box"] = {
    Sheet = sheet,
    Bounds = {
        X = 0,
        Y = 0,
        Width = 1153,
        Height = 350
    }
};
root.SysMesBoxDisplay.Box = name .. "Box";

root.Sprites[name .. "BoxDecoration"] = {
    Sheet = sheet,
    Bounds = {
        X = 1383,
        Y = 117,
        Width = 592,
        Height = 8
    }
};
root.SysMesBoxDisplay.BoxDecoration = name .. "BoxDecoration";

root.Sprites[name .. "SelectionLeftPart"] = {
    Sheet = sheet,
    Bounds = {
        X = 820,
        Y = 666,
        Width = 91,
        Height = 32
    }
};
root.SysMesBoxDisplay.SelectionLeftPart = name .. "SelectionLeftPart";

root.Sprites[name .. "SelectionRightPart"] = {
    Sheet = sheet,
    Bounds = {
        X = 820,
        Y = 666,
        Width = 91,
        Height = 32
    }
};
root.SysMesBoxDisplay.SelectionRightPart = name .. "SelectionRightPart";

root.Sprites[name .. "SelectionMiddlePart"] = {
    Sheet = sheet,
    Bounds = {
        X = 820,
        Y = 666,
        Width = 91,
        Height = 32
    }
};
root.SysMesBoxDisplay.SelectionMiddlePart = name .. "SelectionMiddlePart";